app/javascript/dashboard/modules/search/components/SearchView.vue

```diff
@@ -1,46 +1,62 @@
-      @click="onBack"
-    />
-    <header>
-      <search-header @search="search" />
-      <search-tabs :tabs="tabs" @tab-change="tab => (selectedTab = tab)" />
-    </header>
-    <div class="search-results">
-      <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
-      <div v-else>
-        <div v-if="all.length">
-          <search-result-contacts-list
-            v-if="filterContacts"
-            :contacts="contacts"
-            :query="query"
-          />
-          <search-result-messages-list
-            v-if="filterMessages"
-            :messages="messages"
-            :query="query"
-          />
-          <search-result-conversations-list
-            v-if="filterConversations"
-            :conversations="conversations"
-            :query="query"
-          />
-        </div>
-        <div v-else-if="showEmptySearchResults" class="empty">
-          <fluent-icon icon="info" size="16px" class="icon" />
-          <p class="empty-state__text">
-            {{ $t('SEARCH.EMPTY_STATE_FULL', { query }) }}
-          </p>
+  <div class="backdrop">
+    <section v-on-clickaway="closeSearch" class="search-root">
+      <woot-button
+        color-scheme="secondary"
+        size="large"
+        icon="dismiss"
+        variant="smooth"
+        class="modal--close"
+        @click="onBack"
+      />
+      <header>
+        <search-header @search="search" />
+        <search-tabs
+          v-if="!showEmptySearchResults && all.length"
+          :tabs="tabs"
+          @tab-change="tab => (selectedTab = tab)"
+        />
+      </header>
+      <div class="search-results">
+        <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
+        <div v-else>
+          <div v-if="all.length">
+            <search-result-contacts-list
+              v-if="filterContacts"
+              :contacts="contacts"
+              :query="query"
+              @close-search="closeSearch"
+            />
+            <search-result-messages-list
+              v-if="filterMessages"
+              :messages="messages"
+              :query="query"
+              @close-search="closeSearch"
+            />
+            <search-result-conversations-list
+              v-if="filterConversations"
+              :conversations="conversations"
+              :query="query"
+            />
+          </div>
+          <div v-else-if="showEmptySearchResults && !all.length" class="empty">
+            <fluent-icon icon="info" size="16px" class="icon" />
+            <p class="empty-state__text">
+              {{ $t('SEARCH.EMPTY_STATE_FULL', { query }) }}
+            </p>
+          </div>
+          <div v-else class="empty text-center">
+            <p class="text-center">
+              <fluent-icon icon="search" size="24px" class="icon" />
+            </p>
+            <p class="empty-state__text">
+              {{ $t('SEARCH.EMPTY_STATE_DEFAULT') }}
+            </p>
+          </div>
```

Comment:
This does not trigger correctly when clearing the input.

Steps
1. Search a string that will result an empty search
2. Delete the input

The empty state is the one of "no result found" for the previous search

<img width="582" alt="image" src="https://user-images.githubusercontent.com/18097732/224644856-a36bd354-348b-41ee-915a-d2f4b7685df4.png">